2018/002-update-channels.md

```diff
@@ -0,0 +1,130 @@
+ Title
+
+| Field  | Value      |
+|:-------|:-----------|
+| Status | Draft      |
+| Date   | 2018-07-31 |
+
+## Introduction
+
+### Problem description
+
+Updates can be disruptive for a customer's application stack (ex. Kubernetes
```

Comment:
To clarify, this RFC is only describing the process for RPM based updates, correct? (no image updates from a registry)

Reply:
Correct, it's about updating of RPMs with the channel switch for intrusive updates